app/Services/CampaignService.php

```diff
@@ -62,17 +63,16 @@ public function getCampaignIdsFromSignups()
      */
     public function getPostTotals($campaign)
     {
-        return DB::table('posts')
-                ->select('campaign_id',
-                    DB::raw('SUM(status = "accepted") as accepted_count'),
-                    DB::raw('SUM(status = "pending") as pending_count'),
-                    DB::raw('SUM(status = "rejected") as rejected_count'))
-                ->whereIn('status', ['accepted', 'pending', 'rejected'])
-                // @TODO: if we ever want to expose other post types in Rogue to review/count, update this.
-                ->whereIn('type', ['photo', 'text'])
-                ->where('campaign_id', '=', $campaign['id'])
-                ->whereNull('deleted_at')
-                ->first();
+        return Post::select(
+                'campaign_id',
+                DB::raw('SUM(status = "accepted") as accepted_count'),
+                DB::raw('SUM(status = "pending") as pending_count'),
+                DB::raw('SUM(status = "rejected") as rejected_count'))
+            ->whereIn('status', ['accepted', 'pending', 'rejected'])
```

Comment:
I know this wasn't changed in this PR, but why do we need this line?

Reply:
I'm also interested in this!

Reply:
I'm curious if it was to exclude `voter_reg` posts since the new scope will take care of that 🔭

Reply:
@katiecrane great q! I was confused by this when I was working on this too! But this is to exclude the `voter-reg` statuses :)

Reply:
oh wait - I think I lied because the query is specifically counting `accepted`, `rejected`, and `pending` - thanks for flagging and deleting this now!